public/locales/fr/authentication.json

```diff
@@ -0,0 +1,30 @@
+{
+    "Login": {
+        "username": "Nom d'utilisateur",
+        "password": "Mot de passe",
+        "title": "Connexion",
+        "switchText": "Vous n'avez pas de compte? ",
+        "switchLink": "Inscrivez-vous"
+    },
+    "Signup": {
+        "username": "Nom d'utilisateur",
+        "password": "Mot de passe",
+        "confirmPassword": "Confirmez le mot de passe",
+        "title": "Inscrivez-vous",
+        "switchText": "Vous avez déjà un compte? ",
+        "switchLink": "Connexion",
+        "dataPolicy": "En cochant cette case, je comprends que les données que j'ai saisies seront stockées et ne doivent donc pas être des informations sensibles.",
+        "dataReminder": "Pour rappel, l'adresse e-mail professionnelle, le numéro de téléphone ou le nom réel sont considérés comme des informations sensibles."
+    },
+    "Errors": {
+        "username": "Le nom d'utilisateur est requis",
+        "password": "Le mot de passe est requis",
+        "confirmPassword": "La confirmation du mot de passe est requise",
+        "passwordMatch": "Les mots de passe ne correspondent pas",
+        "usernameTaken": "Le nom d'utilisateur est déjà pris",
+        "notFound": "Impossible de trouver la ressource, le backend est probablement en panne",
+        "unknown": "Une erreur inconnue s'est produite",
+        "server": "Erreur du serveur",
+        "unauthorized": "Nom d'utilisateur ou mot de passe invalide"
+    }
+}
```

Comment:
Please mind the EOFs. Never have to worry about it again: https://stackoverflow.com/a/44704969

Reply:
yikes my bad, I'll run prettier anyways which will add those eof